data.py

```diff
@@ -5,27 +5,50 @@
 # This source code is licensed under the license found in the
 # LICENSE file in the root directory of this source tree.
 
+import logging
 import math
 
 import torch
 import torch.nn as nn
 import torchaudio
-import logging
-
-from models.multimodal_preprocessors import SimpleTokenizer
 from PIL import Image
 from pytorchvideo import transforms as pv_transforms
 from pytorchvideo.data.clip_sampling import ConstantClipsPerVideoSampler
 from pytorchvideo.data.encoded_video_pyav import EncodedVideoPyAV
-
 from torchvision import transforms
-from torchvision.transforms._transforms_video import NormalizeVideo
+
+from models.multimodal_preprocessors import SimpleTokenizer
 
 DEFAULT_AUDIO_FRAME_SHIFT_MS = 10  # in milliseconds
 
 BPE_PATH = "bpe/bpe_simple_vocab_16e6.txt.gz"
 
 
+class NormalizeVideo:
```

Comment:
Is it similar to https://torchvideo.readthedocs.io/en/latest/transforms.html#torchvideo.transforms.NormalizeVideo?

Reply:
yes, but it's a bit wasted to introduce a whole package for one function, I found we could use `pytorchvideo.transforms.Normalize`, which is already imported. I have tested there output are equal.